bin/run-addon

```diff
@@ -12,7 +12,9 @@ firefoxes="
 /Applications/FirefoxNightly.app
 /Applications/FirefoxDeveloperEdition.app
 /Applications/FirefoxAurora.app
+/usr/bin/firefox
 "
+# TODO add more firefoxes locations? use `which` maybe?
```

Comment:
Adding `$(which firefox)` to this list should work, but there's a possibility if it's not found that it will cause the script to error, in which case you'd want to use `$(which firefox || true)` (to swallow the error).  But please test first, I'm not sure what errors in scripts in string substitutions do.